gcloud/monitoring/client.py

```diff
@@ -195,6 +202,126 @@ def metric_descriptor(self, type_,
             display_name=display_name,
         )
 
+    @staticmethod
+    def metric(type_, labels):
+        """Factory for constructing metric objects.
+
+        :class:`~gcloud.monitoring.metric.Metric` objects are typically
+        created to write custom metric values. The type should match the
+        metric type specified in the
+        :class:`~gcloud.monitoring.metric.MetricDescriptor` used to
+        create the custom metric::
+
+             >>> metric = client.metric('custom.googleapis.com/my_metric',
+             ...                        labels={
+             ...                            'status': 'successful',
+             ...                         })
+
+        :type type_: string
+        :param type_: The metric type name.
+
+        :type labels: dict
+        :param labels: A mapping from label names to values for all labels
+                       enumerated in the associated
+                       :class:`~gcloud.monitoring.metric.MetricDescriptor`.
+
+        :rtype: :class:`~gcloud.monitoring.metric.Metric`
+        :returns: The metric object.
+        """
+        return Metric(type=type_, labels=labels)
+
+    @staticmethod
+    def resource(type_, labels):
+        """Factory for constructing monitored resource objects.
+
+        Creating a monitored resource (
+        :class:`~gcloud.monitoring.resource.Resource`) object. A monitored
```

Comment:
One idea would be to leave it out and add "(a `Resource` instance)" after "monitored resource object" in the following sentence.